qurro/_json_utils.py

```diff
@@ -360,3 +360,25 @@ def replace_js_json_definitions(
     # If this was called on a JS test then this perfectly normal, but if this
     # was called during the execution of Qurro then this is a problem.
     return 1
+
+
+def check_json_dataset_names(json_dict, *restricted_names):
+    """Checks that certain dataset names aren't present in a Vega-Lite JSON.
+
+       This should never really fail, since all of the special dataset names we
+       use internally start with "qurro_" (and Altair generates hashes of the
```

Comment:
I want this function to send me an email if it ever happens. I would love to know the context. haha

Reply:
SAME